tests/common.bash

```diff
@@ -371,18 +371,29 @@ log_level = "debug"
 EOF
 }
 
-function install_kata() {
-	local kata_tarball="kata-static.tar.xz"
-	declare -r katadir="/opt/kata"
+function install_kata_common() {
+	declare -r katadir="$1"
 	declare -r destdir="/"
-	declare -r local_bin_dir="/usr/local/bin/"
+	declare -r kata_tarball="kata-static.tar.xz"
 
 	# Removing previous kata installation
 	sudo rm -rf "${katadir}"
 
 	pushd "${kata_tarball_dir}"
 	sudo tar -xvf "${kata_tarball}" -C "${destdir}"
 	popd
+}
+
+function install_kata_tools() {
+	declare -r katadir="/opt/kata"
+	install_kata_common "${katadir}"
```

Comment:
I suppose you will change this when tools are split out of the kata tarball (https://github.com/kata-containers/kata-containers/issues/8864 ). Right? Worth a `TODO` comment?